manifest.firefox.json

```diff
@@ -1,7 +1,7 @@
 {
     "manifest_version": 3,
     "name": "TestRail+",
-    "version": "1.0.0",
+    "version": "2.0.3",
```

Comment:
We really need to automate this based on the tag using something like `git describe` otherwise I can guarantee we'll forget at some point and it will be embarrassing.

Reply:
The pipeline now updates manifest.json and package.json to the latest tag